pkgs/shells/marcel/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, bash
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "marcel";
+  version = "0.17.0";
```

There has been a `v0.20.0` release yesterday, which fixed most of the hard-coded `/bin/bash` stuff, so that no longer needs patching.
Though, there hasn't been an actual GitHub release, only a tag. So until there's a release, you can use `rev="refs/tags/v${version}"` to fetch by tag.

(When changing versions, don't forget to change/delete the hash, else it will still use the previous version)
```suggestion
  version = "0.20.0";
```